nengo/_vendor/scipy/sparse/nonscipy_utils.py

```diff
@@ -0,0 +1,10 @@
+import math
+
+# simple custom implementation of n choose k
+def comb(n, k):
+    """Compute n choose k"""
+    if k > n or k < 0:
+        return 0
+    if k == 0:
+        return 1
+    return math.factorial(n) // math.factorial(n - k) // math.factorial(k)
```

Comment:
Where did this implementation come from? For scipy when `exact=False` (the default) it uses this implementation: https://github.com/scipy/scipy/blob/v1.3.0/scipy/special/_comb.pyx#L7

Reply:
I wrote it. I'm fine to use the other one.

Reply:
Done in #1535